src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -265,7 +265,7 @@ void EnOssan_Init(Actor* thisx, PlayState* play) {
     EnOssan* this = THIS;
     s16 id;
 
-    //! @bug Condition is impossible, params cannot be both greater then 1 AND less then 0.
+    //! @bug Condition is impossible, && should be ||
```

Comment:
There's another one in z_en_dnp.c that is inside the conditional scope, we should decide which side they should go to be consistent. I'm in two minds about removing the extra detail as well, since it's not clear what the problem is without it (which could be the thing that caused the bug in the first place).